src/jit/lower.cpp

```diff
@@ -240,6 +240,22 @@ GenTree* Lowering::LowerNode(GenTree* node)
 
         case GT_STORE_BLK:
         case GT_STORE_OBJ:
+            if (node->AsBlk()->gtBlkSize == 0)
```

Comment:
Can OBJ have size 0? Seems like it should never happen.

Reply:
Yes, that's what I figured, but I'm not absolutely certain, and I didn't think the distinction was worth the time expenditure to track it down.
Thanks, BTW for pointing out the issues on the previous PR & test. I wanted to get the fix for the cpBlk bug in by last Friday's snap to the preview branch.